Tests/WinAppDriverTests/AppDriverOptionsTest.swift

```diff
@@ -0,0 +1,34 @@
+import TestsCommon
+import WinSDK
+import XCTest
+
+@testable import WebDriver
+@testable import WinAppDriver
+
+class AppDriverOptionsTest: XCTestCase {
+    func tempFileName() -> String {
+        return FileManager.default.temporaryDirectory.appendingPathComponent(UUID().uuidString)
+            .appendingPathExtension("txt").path
+    }
+
+    /// Tests that redirecting stdout to a file works.
+    func testStdoutRedirectToFile() throws {
+        let outputFile = try {
```

Comment:
Why this block?

Reply:
So we wait for the `MSInfo32App` to finish. There is probably a better way of doing this but my Swift knowledge is still very limited.

Reply:
I believe it's identical to:
```swift
let outputFile = tempFileName()
let _ = try MSInfo32App(
    winAppDriver: WinAppDriver.start(
         outputFile: outputFile))

Reply:
There's no asynchrony involved

Reply:
Ack, will follow-up.